src/Talis/Persona/Client/Base.php

```diff
@@ -345,21 +344,30 @@ protected function performRequest($url, array $opts)
         try {
             $response = $request->send();
         } catch(RequestException $exception) {
-            if (isset($exception->hasResponse) && $exception->hasResponse) {
-                $status = $exception->getResponse()->getStatusCode();
-            } else {
+            $response = $exception->getRequest()->getResponse();
+            if(isset($response))
```

Comment:
I saw something in the Guzzle docs about a call to `hasResponse()` - should we be using that? http://docs.guzzlephp.org/en/latest/quickstart.html#exceptions